src/test/java/hudson/plugins/throttleconcurrents/ThrottleIntegrationTest.java

```diff
@@ -80,8 +84,19 @@ private void setupSlave() throws Exception {
         slave.setRetentionStrategy(waterMark);
     }
     
+    /**
+     * setup security so that no one except SYSTEM has any permissions.
+     * should be called after {@link #setupSlave()}
+     */
+    private void setupSecurity() {
+        jenkins.setSecurityRealm(createDummySecurityRealm());
+        GlobalMatrixAuthorizationStrategy auth = new GlobalMatrixAuthorizationStrategy();
+        jenkins.setAuthorizationStrategy(auth);
+    }
+    
```

Comment:
The problem is a permission issue, and requires no READ permission for `ANONYMOUS` to reproduce.

Creating a slave triggers serializing Jenkins configuration and results an exception as `DummySecurityRealm` is not capable to be serialized.
So you have to call `setupSecurity` after `setupSlave`.